media/css/protocol/components/_sub-navigation.scss

```diff
@@ -165,3 +175,24 @@
         }
     }
 }
+
+// * -------------------------------------------------------------------------- */
+// Overrides to align with subnav https://github.com/mozilla/bedrock/issues/14081
+
+    nav.mzp-c-breadcrumb {
```

Comment:
At this point, I believe it would make more sense to have these override style changes be a part of the Protocol component : )

Edit: Just viewed [your comment above](https://github.com/mozilla/bedrock/pull/14413#discussion_r1615283661) and looks like we agree